src/GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests/Services/ImporterLocationServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Data.Processor.Services;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Database.StatisticsDbUtils;
+namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests.Services
+{
+    public class ImporterLocationServiceTests
+    {
+        [Fact]
+        public async Task Find_EnglishDevolvedAreaCaseSensitiveLocationName()
+        {
+            var country = new Country("1", "england");
+            var lowerCase = new EnglishDevolvedArea("2", "casesensitive");
+            var upperCase = new EnglishDevolvedArea("1", "CASESENSITIVE");
+            var statisticsDbContextId = Guid.NewGuid().ToString();
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(statisticsDbContextId))
+            {
+                var service = BuildImporterLocationService();
+                var result1 = service.Find(statisticsDbContext, country,
```

Comment:
I think it would be worth sticking in a link to https://github.com/dotnet/efcore/issues/6153 in a comment to say this test is not actually testing what it should be due to InMemory database being case sensitive rather than case insensitive like SqlServer.